src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,92 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Browsing Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Applications menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+<video autoplay loop>
+    <source src="/images/locate-launch-applications/drag-application.webm" />
+</video>
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop, then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, the Applications menu, and from the Terminal.
+
+### Using the Launcher
+
+Press the `SUPER` key or the Show Launcher icon to bring up the Launcher.
+
+![Show Launcher](/images/locate-launch-applications/show-launcher.png)
+
+Search for the name of the application, then hit `Enter`.
+
+![Search App Name](/images/locate-launch-applications/search-app.png)
+
+### Using the Applications Menu
+
+Click the Applications menu icon located in the Dock.
+
+![Click Applications Dock](/images/locate-launch-applications/click-applications-dock.png)
+
+Click the application's icon to launch it.
+
+![Click App Icon](/images/locate-launch-applications/click-app-icon.png)
+
+### Using the Dock
+
+Click the application's icon pinned to the Dock.
+
+![Dock Icon](/images/locate-launch-applications/dock-icon.png)
+
+You can pin additional applications to the Dock by right-clicking the application's icon in the Application menu and selecting `Add to Favorites`.
+
+![Add to Favorites](/images/locate-launch-applications/add-to-favorites.png)
+
+Remove applications from the Dock by right-clicking the application and selecting `Remove from Favorites`.
+
+![Remove from Favorites](/images/locate-launch-applications/remove-from-favorites.png)
+
+### Using Alt + F2
+
+Press `Alt` + `F2` to bring up a command entry window. Enter the name of the application and press `Enter`.
+
+![Command Launcher](/images/locate-launch-applications/command-launcher.png)
+
+### Using the Terminal
+
+Launch the Terminal from the dock; then type the name of the application hit `Enter`. You can also launch the Terminal by pressing `SUPER` + `T`.
```

Comment:
I don't know how I missed this before, but the part of the first sentence after the semicolon would be missing a word (e.g. "and" or "then"). I also think this is an unnecessary usage of the semicolon, similar to line 30/32 earlier. It can instead be:

> Launch the Terminal from the dock, then type the name of the application and hit `Enter`.

Additionally, I would combine the two sentences together so the methods to launch the terminal are grouped together:

> Launch the Terminal from the dock or by pressing `SUPER` + `T`, then type the name of the application and hit `Enter`.

Otherwise, by the time the user's gotten to the end of the first sentence, they've already launched the terminal and backtracking to the second way to launch the terminal is sort of off-topic.